HOWTORELEASE

```diff
@@ -0,0 +1,54 @@
+HOWTO RELEASE THE PROJ-DATUMGRID PACKAGE
+========================================
+
+Prerequisites:
+
+    - PROJ 4.9.3 or newer. Explained here why:
+      http://lists.maptools.org/pipermail/proj/2016-October/007544.html
+    - CMake/CPack
+
+1. Adjust the version number:
+
+    If preparing a release candidate "RCx" should be added
+    to PROJ_DATUMGRID_VERSION_MINOR in CMakeLists.txt.
+
+    If preparing the final release any postfixes to the version
+    number should be stripped.
+
+2. Create tarball and zip:
+
+    mkdir build
+    cd build
+    cmake ..
+    make dist
+
+3. Check repo for grid changes:
+
+    git status
+
+4. Check contents of tarball and zip
+
+    tar tvf proj-datumgrid-1.7RC1.tar.gz
+    unzip -l proj-datumgrid-1.7RC1.zip
```

Comment:
`unzip -l proj-datumgrid-*.zip` is better, no need to hardcode the expected version.